SECURITY.md

```diff
@@ -0,0 +1,28 @@
+# Security Policy
+
+Thank you for taking the time to report a security vulnerability. We would like to investigate every report thoroughly.
+
+## Reporting a Vulnerability
+
+To make a report, please send an email to the `mail ID` with the vulnerability details. In case you would like to submit any other kind of issue, please open one in the appropriate repository.
```

Comment:
How do we want to report? Slack might be easiest?

Reply:
Yes it will be but it won't be discreet and will defeat the purpose of asking them to separately report. Or, we'll have to give one specific person's slack to dm.

Reply:
Slack DM I think. Give some time for people to discuss I think